cms/static/js/views/paging_header.js

```diff
@@ -37,13 +37,11 @@ define(["underscore", "gettext", "js/views/baseview"], function(_, gettext, Base
                 if (this.view.collection.sortDirection === 'asc') {
                     // Translators: sample result:
                     // "Showing 0-9 out of 25 total, filtered by Images, sorted by Date Added ascending"
-                    message = gettext('Showing %(current_item_range)s out of %(total_items_count)s, ' +
-                        'filtered by %(asset_type)s, sorted by %(sort_name)s ascending');
+                    message = gettext('Showing %(current_item_range)s out of %(total_items_count)s, filtered by %(asset_type)s, sorted by %(sort_name)s ascending');
```

Comment:
@martynjames - FYI this is a translation string you broke. You cannot do this - the strings won't scrape properly.

You MUST, you absolutely MUST, test your translations before merging. Instructions on how to do that can be found at edx.readthedocs.org/projects/edx-developer-guide/en/latest/internationalization/i18n.html#coverage-testing

cc @smagoun